hadoop-hdds/interface-server/src/main/proto/ScmServerDatanodeHeartbeatProtocol.proto

```diff
@@ -390,6 +404,14 @@ message ClosePipelineCommandProto {
   required int64 cmdId = 2;
 }
 
+/**
+ * This command asks the DataNode to finalize a new HDDS layout version.
+ */
+message FinalizeNewLayoutVersionCommandProto {
+  required bool finalizeNewHDDSLayoutVersion = 1 [default = false];
```

Comment:
What is the use case the command where this flag=false. Is it a no-op request?

Reply:
yes, that would be a no op request.

Reply:
Is there any use case for this noop request? Will it be used by somebody? Why do we need this field?